src/main/java/uk/nhs/tis/trainee/notifications/model/History.java

```diff
@@ -52,7 +53,9 @@ public record History(
     Instant readAt,
     NotificationStatus status,
     String statusDetail,
-    Instant lastRetry) {
+    Instant lastRetry,
+    Object testObjectDate,
+    LocalDate testLocalDate) {
```

Comment:
The custom writer is called for both testObjectDate and testLocalDate, but the custom reader is only called for testLocalDate.